### PR TITLE
jenkins: add tab exception for Makefile

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

File: jenkins/check_tabs.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ rm -f $ERRFILE
 git diff --name-only `git merge-base origin/master HEAD` HEAD |
     xargs grep -P -l "\t" |
     grep -v -E '^pal/' |
+    grep -v -E '\Makefile$' |
```
Comment (dilijev) [inline jenkins/check_tabs.sh]: > `/M`

Escaping the `M`? Did you mean `^`?

**Comment by dilijev** [comment]
Weird... why are so many unrelated files failing the EOL Check?

File: jenkins/check_tabs.sh (hunk 1 above)
Comment (obastemur) [inline jenkins/check_tabs.sh]: This should just work though.  `\M` -> `M`

**Comment by obastemur** [comment]
> Weird... why are so many unrelated files failing the EOL Check?

https://github.com/Microsoft/ChakraCore/pull/2875

**Comment by dilijev** [comment]
@obastemur ah, thanks. Figured it was from a manual merge.

**Comment by dilijev** [comment]
@dotnet-bot 
test EOL Check
test Ubuntu ubuntu_linux_debug

**Comment by dilijev** [comment]
Checks passed. LGTM

File: jenkins/check_tabs.sh (hunk 1 above)
Comment (dilijev) [inline jenkins/check_tabs.sh]: Sure. Just wondering what the intent here was.

**Comment by dilijev** [comment]
Checks passed. LGTM except possibly resolving this: https://github.com/Microsoft/ChakraCore/pull/2874#discussion_r113802380

**Comment by obastemur** [comment]
@dilijev Thanks for the review

**Pull request merged**